src/socketx.h

```diff
@@ -3,6 +3,7 @@
 #include <winsock.h>
 typedef int socklen_t;
 #pragma warning(disable: 4127) // incompatible with FD_SET()
+#define socketerror  WSAGetLastError()
```

Comment:
Use `#define socketerror()` so the use of this doesn't look like a variable.

Reply:
Fixed.

Reply:
This has not been fixed.